src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx

```diff
@@ -116,7 +116,7 @@ export default function TerminalView({
             return true
         })
     }
-
+    // TODO: Verify
```

Comment:
Is it done?

Reply:
removed the condition and did sanity for the same